onyo/shell_completion/zsh/_onyo

```diff
@@ -121,6 +121,7 @@ _onyo() {
                 args+=(
                     '(- : *)'{-h,--help}'[show this help message and exit]'
                     '(-t --template)'{-t,--template}'[template to seed the new assets]:TEMPLATE:_path_files -W "$(_template_dir)" -g "*(.)"'
```

Comment:
This line also should be adjusted to exclude `-c` and `--clone`.

Reply:
That should've been obvious. Thanks!

Reply:
Pushed